hips/utils/tests/test_healpix.py

```diff
@@ -22,3 +24,17 @@ def test_boundaries():
     radec_precomp = [[264.375,  258.75,  264.375,  270.],
                      [-24.624318,  -30.,  -35.685335,  -30.]]
     assert_allclose([radec.ra, radec.dec], radec_precomp)
+
+def test_compute_healpix_pixel_indices():
+    order = 3
+    nside = hp.order2nside(order)
+
+    skycoord = SkyCoord(10, 20, unit="deg")
+    wcs_geometry = WCSGeometry.create(skydir=skycoord, shape=(10, 20), coordsys='CEL', projection='AIT', cdelt=1.0, crpix=(1., 1.))
+
+    pixels_precomp = [84, 111, 112, 113]
+    len_precomp = 39
```

Comment:
For cases like here where you just define a local variable `len_precomp` and use it once on a line that isn't very long, I'd suggest o just use the value in the assert, i.e. not introduce the local variable.

Personally I do introduce local variables to make code more readable when lines get too long. But in cases like here where the assert statement is very simple and the line is short, no need for the local variable.